traffic-id.yaml

```diff
@@ -102,6 +117,22 @@ tls-sni-patterns:
       - feelinsonice-hrd.appspot.com
       - snapchat.com
 
+  # Spotify
+  - id: spotify
+    labels:
+      - music
```

Comment:
Should Spotify be streaming as well?

Reply:
wonder if we need a 'streaming' as well as 'video' and 'music' in general?

Reply:
Or should we use "streaming-music" and "streaming-video"?